docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+    "        \"multiannotator_labels\": s[row_NA_check].reset_index(drop=True),\n",
+    "    }\n",
+    "\n",
+    "data_dict = make_data()\n",
+    "\n",
+    "X = data_dict[\"X_train\"]\n",
+    "multiannotator_labels = data_dict[\"multiannotator_labels\"]\n",
+    "true_labels = data_dict[\"true_labels_train\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e95d31b2",
+   "metadata": {},
+   "source": [
+    "For this tutorial we are using a toy dataset that has 50 annotators and 160 examples. There are three possible classes, `0`, `1` and `2`.\n",
+    "\n",
+    "Each annotator annotates approximately 10% of the examples. We also synthetically made the last 5 annotators in our toy dataset have much noisier labels than the rest of the annotators."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "946726ad",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Bringing Your Own Data (BYOD)?\n",
+    "\n",
+    "You can easily replace the above with your own multiannotator dataset, and continue with the rest of the tutorial.\n",
+    " \n",
+    "`multiannotator_labels` should be a numpy array or pandas DataFrame with each column representing an annotator and each row representing an example. Your classes (and entries of `multiannotator_labels`) should be represented as integer indices 0, 1, ..., num_classes - 1, where examples that are not annotated by a particular annotator are represented using `np.nan`.\n",
+    "\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51335def",
+   "metadata": {},
+   "source": [
+    "## 3. Get majority vote label and compute out-of-sample predicted probabilites"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1857cc7",
+   "metadata": {},
+   "source": [
+    "Before training our machine learning model, we must first obtain the consensus labels from the annotators that labeled the data. The simplest way to obtain an initial set of consensus labels is to select it using majority vote."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d009f347",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "majority_vote_label = get_majority_vote_label(multiannotator_labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7287b733",
+   "metadata": {},
+   "source": [
+    "Next, we will train our model on the consensus labels obtained using majority vote to compute out-of-sample predicted probabilities. Here, we use a simple logistic regression model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cbd1e415",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.linear_model import LogisticRegression\n",
+    "from sklearn.model_selection import cross_val_predict\n",
+    "\n",
+    "model = LogisticRegression()\n",
+    "\n",
+    "num_crossval_folds = 5  \n",
+    "cv_pred_probs = cross_val_predict(\n",
```

Comment:
maybe better to just call this `pred_probs` since that's our canonical name for it throughout the codebase?

Reply:
That makes sense! I used `cv_pred_probs` because I saw it in some of our other tutorials ([audio](https://docs.cleanlab.ai/v2.0.0/tutorials/audio.html) and [workflow overviews](https://docs.cleanlab.ai/v2.0.0/tutorials/indepth_overview.html)). I can open a separate PR to change those